### PR TITLE
Sending data

### PR DESCRIPTION
**Pull request by MisterJames**

### CODE REVIEW

File: content/asp.net/getting-started/sending-data.md [hunk 1]
```diff
@@ -1,20 +1,100 @@
...
+In the above example, each friend selected in a checkbox and submitted to the action will be in the `friends` parameter.
+
+Another point to make with the binding process is that you essentially get one opportunity to bind parameter values using any part of the `BODY`. The `BODY` is a stream that can be read from, but you cannot rely on being able to rewind the stream and re-read it. For this reason, a maximum of one parameter may contain a binding attribute that reads from the `BODY` of the request.
 
 ## Next Steps
 
-Give the reader some additional exercises/tasks they can perform to try out what they've just learned.
\ No newline at end of file
+ - Review the [routing](routing.md) concepts to understand how to extract tokens that can be used in the model binding process.
+ - Have a look again at [configuration](configuration.md) or read ahed into [dependency injection](controller-dependencies.md) if you are interested in getting environment- or application-level data into your controller.
```
Comment (ardalis) [inline content/asp.net/getting-started/sending-data.md]: "read ahed" -> "read ahead"

File: content/asp.net/getting-started/sending-data.md [hunk 2]
```diff
@@ -1,20 +1,100 @@
...
+ - Experiment with the sample project to see the different model binding in action. Be sure to set breakpoints so that you can inspect the values of the parameters as your action is entered.
```
Comment (ardalis) [inline content/asp.net/getting-started/sending-data.md]: "see the different model binding in action" -> "see the different kinds of model binding in action"

File: content/asp.net/getting-started/sending-data.md [hunk 3]
```diff
@@ -1,20 +1,100 @@
...
 
+    http://portal.aplineskihouse.com/user/view/100
+
+A body of the request may optionally contain a `form-data` section of the request. This allows for another key-value transport mechanism for the client, typically the method of choice for default HTML form submission.
+
+Form aside, the `BODY` of the request may come more into play specifically when you're working with HTTP verbs like `POST` or `PUT` and using non-HTTP formats. The body can be more expressive than just the `form-data` as it can contain payload of your design, and it can follow standard object notation such as XML or JSON.
+
+By nature and by virtue HTTP and HTML do not provide a strongly-typed context that a language can make use of. These data sources are simple, text-based inputs to our application and are typically thought of as key-value pairs. Without some kind of support, you would need to manually inspect all of the sources, extract the values and then cast them to the appropriate object types in your application. This is where "model binding" comes in to play, and makes light work of the task of working with the key-value pairs.
```
Comment (MisterJames) [inline content/asp.net/getting-started/sending-data.md]: heheh...that's a good one :)
"I am coming in to play."

Comment (ardalis) [inline content/asp.net/getting-started/sending-data.md]: My thought exactly. :)

**Pull request merged**